proto/celestia/minfee/v1/query.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package celestia.minfee.v1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/celestiaorg/celestia-app/x/minfee";
+
+// Query defines the gRPC querier service.
+service Query {
+  // NetworkMinGasPrice queries the network wide minimum fee
```

Comment:
I interpret fee as gasPrice * gas so IMO gas price is more correct here

```suggestion
  // NetworkMinGasPrice queries the network wide minimum gas price.
```

if we wanted to be really verbose it could say

```suggestion
  // NetworkMinGasPrice queries the network wide minimum gas price per unit of gas.
```